packages/remix-dev/cli/migrate/migrations/convert-to-javascript/index.ts

```diff
@@ -34,16 +34,20 @@ export const convertToJavaScript: MigrationFunction = async ({
     transformPath: TRANSFORM_PATH,
   });
   if (!codemodOk) {
-    console.error("❌ I couldn't update your imports to JS.");
-    if (!flags.debug) {
-      console.log("👉 Try again with the `--debug` flag to see what failed.");
+    if (!flags.silent) {
```

Comment:
I think if there's an error we should have output even if silent is set to true right?

Reply:
The reason why I want to hide the `👉 Try again with the `--debug` flag to see what failed.` message, is that we're running the migration programmatically instead of via the `migrate` CLI

Reply:
I see what you mean. Maybe we should do some code duplication then. Seems like we have two different use cases for this code.

Reply:
That would mean a lot of duplication (like almost the whole folder) I'm afraid 😕

Reply:
`--no-interactive` is a common flag for doing that sort of thing, so we could use that

Reply:
Or actually, even better, we can auto-detect if we're running via command line or as a function via `require.main === module`. see https://stackoverflow.com/a/6398335/1490091

Reply:
That sounds great @pcattori 👍

Reply:
Just to be clear: I should put the `isInteractive` check _only_ around the `--debug` log, right?

Reply:
Yep